packages/react/src/components/SideMenu/SideMenuWrapper.tsx

```diff
@@ -0,0 +1,8 @@
+import { Group } from "@mantine/core";
+import { ReactNode } from "react";
+
+export const SideMenuWrapper = (props: { children: ReactNode }) => (
```

Comment:
what's the use of SideMenuWrapper vs just putting this code in SideMenu.tsx?

Reply:
It's the side menu equivalent of `FormattingToolbarWrapper`, so it's just a basic parent element which keeps the default styling. It's useful when you're replacing the items so that you don't have to re-style the menu. Same thing with `DragHandleWrapper`.

Reply:
Might be worth to rename `SuggestionMenuWrapper` instead, because it's used quite differently to `ToolbarWrapper`, `SideMenuWrapper`, and `DragHandleMenuWrapper`.